sdk/java-sdk/src/main/scala/kalix/javasdk/impl/valueentity/ValueEntitiesImpl.scala

```diff
@@ -167,12 +164,12 @@ final class ValueEntitiesImpl(system: ActorSystem, val services: Map[String, Val
 
           val serializedSecondaryEffect = effect.secondaryEffect match {
             case MessageReplyImpl(message, metadata, sideEffects) =>
-              MessageReplyImpl(service.anySupport.encodeJava(message), metadata, sideEffects)
+              MessageReplyImpl(service.messageCodec.encodeScala(message), metadata, sideEffects)
```

Comment:
We have been encoding with Java and then converting to Java, like in: `ScalaPbAny.fromJavaProto(service.anySupport.encodeJava(newState)`

This was the only place where we were not wrapping. Just using Scala now, should be ok.